### ISSUE TITLE
Unable to process log entry: value too long for type character varying(200)

### ISSUE THREAD

**Issue by midiotthimble** [getsentry/sentry#1182]
```python
Traceback (most recent call last):
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/sentry/manager.py", line 414, in save_data
    **group_kwargs
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/sentry/manager.py", line 490, in _create_group
    defaults=kwargs
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/django/db/models/manager.py", line 146, in get_or_create
    return self.get_query_set().get_or_create(**kwargs)
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/django/db/models/query.py", line 491, in get_or_create
    obj.save(force_insert=True, using=self.db)
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/sentry/models/group.py", line 90, in save
    super(Group, self).save(*args, **kwargs)
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/django/db/models/base.py", line 546, in save
    force_update=force_update, update_fields=update_fields)
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/django/db/models/base.py", line 650, in save_base
    result = manager._insert([self], fields=fields, return_id=update_pk, using=using, raw=raw)
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/django/db/models/manager.py", line 215, in _insert
    return insert_query(self.model, objs, fields, **kwargs)
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/django/db/models/query.py", line 1675, in insert_query
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 943, in execute_sql
    cursor.execute(sql, params)
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/django/db/backends/postgresql_psycopg2/base.py", line 58, in execute
    six.reraise(utils.DatabaseError, utils.DatabaseError(*tuple(e.args)), sys.exc_info()[2])
  File "/usr/home/sentry/.virtualenvs/sentry/lib/python2.7/site-packages/django/db/backends/postgresql_psycopg2/base.py", line 54, in execute
    return self.cursor.execute(query, args)
DatabaseError: value too long for type character varying(200)
```

`pip list` output: http://bpaste.net/show/290113/

**Comment by cosgroveb** [comment]
Just ran into this today, running version 6.4.4. Was trying to stand up Sentry for a Rails project and are dual writing to emails directly from our app, and to Sentry. We noticed that almost none of the emailed exceptions the app is sending are making it in to Sentry....

After messing around with the django config for some time to get any sort of logging working we noticed this `DatabaseError: value too long for type character varying(200)`. Turning statement logging on in our database we were able to determine that the problem field is `view` in the `sentry_message` and `sentry_groupedmessage` tables is set to `character varying(200)`. 

Worrying is that it was quite a feat to determine that Sentry was choking on our messages and why.

We were able to get everything working in our test environment by changing the column type to text but obviously we won't be doing this in production. Is it possible for the Sentry team to take a look at this and get it fixed?
